app/components/Blockchain/Transaction/Transaction.jsx

```diff
@@ -88,7 +93,15 @@ export default class Transaction extends React.Component<Props> {
 
   handleViewTransaction = () => {
     const { networkId, explorer, tx, chain } = this.props
-    const { txid } = tx
+    let txid
+    switch (chain) {
+      case 'neo3':
+        txid = tx.hash.substring(2)
+        break
+      default:
+        ;({ txid } = tx)
+    }
```

Comment:
This reads and looks a little strange maybe we could just have:
  let txid = tx.txid
  if (chain === 'neo3') txid = tx.hash.substring(2)

Reply:
resolved in 
8063583